back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -20,7 +20,7 @@ def index
 
     @projects = Project.where(id: publications.select(:publication_id))
       .ordered
-      .includes(:project_images, :phases, :areas, admin_publication: [:children])
+      .includes(:project_images, :areas, :topics, :content_builder_layouts, phases: [:report], admin_publication: [:children])
```

Comment:
Aware I'm leaking some engine stuff here, but it's rather urgent and feels like it's not worth the extra complexity of trying to delegate this responsibility to the engine.